### ISSUE TITLE
test_canonical fails when running tests from sdist

### ISSUE THREAD

**Issue by jayvdb** [jfinkhaeuser/prance#45]
https://github.com/jfinkhaeuser/prance/blob/master/tests/specs/symlink_test is supposed to be a symlink, but in the sdist uploaded to PyPI it isnt a symlink.

**Comment by jfinkhaeuser** [comment]
Makes sense. I'll see what I will do. The test is intended to detect whether symlinks are correctly resolved, so simply disabling it isn't going to be a good answer. There may not be a better one, though.

**Comment by jayvdb** [comment]
Dynamically create the symlink in the test ?

**Comment by jfinkhaeuser** [comment]
That smacks a bit of testing your test environment. 1 == 1 will always be true, you know? It may be better than disabling the test, but doesn't quite feel right, either.

**Comment by jayvdb** [comment]
It is just test setup. the symlink is needed to verify that the library dereferences the symlink properly.
